### PR TITLE
Add Playbook for Planning Retreat

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

File: retreat-playbook.md [hunk 1]
```diff
@@ -0,0 +1,481 @@
...
+
+| Time          | Length | Description                                                                |
+|---------------|--------|----------------------------------------------------------------------------|
+| 00:00 - 00:15 | 15mins | [Introductions](#introductions) |
+| 00:15 - 01:00 | 45mins | [5-minute sharebacks of solo visioning and readings](#5-minute-sharebacks) |
+| 01:00 - 01:15 | 15mins | Break!                                                                     |
+| 01:15 - 02:30 | 75mins | [Co-drafting vision & mission template](#Co-drafting-vision-amp-mission)   |
+| 02:30 - 03:00 | 30mins | [Finding value alignment](#Finding-value-alignment)                        |
+
+### Introductions
```
Comment (dcwalk) [inline retreat-playbook.md]:
```suggestion
### Introductions

*Facilitator: Dawn*  
*Scribe: Ben (ensure how we work is documented)*

- How we got here
- What the retreat is for and our aims for today
- Logistics
    - Recording this meeting
    - [Shared notepad](https://hackmd.io/PbNVbPUhSTKL7oEtnrzpuw)
    - Making decisions will be through full consensus of those present
- How we will work together, proposed group agreements:
    - Shared responsibility for conversation (step up listening, step up speaking)
    - One person speaking at a time
    - Honest respect with each other
    - Focused, on topic discussion
    - Keep each other on time
    - Device-free
```

Comment (dcwalk) [reply retreat-playbook.md]: @benhylau -- here are my proposed group agreements taken from previous facilitation work! Updated in the online playbook!

Comment (benhylau) [reply retreat-playbook.md]: Thanks!

**Comment by benhylau** [comment]
No moar changes!

**Pull request merged**